### ISSUE TITLE
dg3c and dpga tests add almost half an hour to CI

### ISSUE THREAD

**Issue by eric-wieser** [pygae/clifford#315]
From the no-jit run:
```
========================== slowest 25 test durations ===========================
395.32s call     clifford/test/test_dg3c.py::BasicTests::test_up_down
305.63s call     clifford/test/test_dpga.py::TestBasicDPGA::test_line
138.70s call     clifford/test/test_dpga.py::TestBasicDPGA::test_rotate
107.38s call     clifford/test/test_dg3c.py::BasicTests::test_up_down_cga1
100.45s call     clifford/test/test_dg3c.py::GeometricPrimitiveTests::test_rotation
80.05s call     clifford/test/test_dpga.py::TestBasicDPGA::test_translate
52.18s call     clifford/test/test_dg3c.py::GeometricPrimitiveTests::test_bivector_orthogonality
44.94s call     clifford/test/test_dpga.py::TestBasicDPGA::test_up_down
37.76s call     clifford/test/test_dg3c.py::GeometricPrimitiveTests::test_translation
32.41s call     clifford/test/test_dg3c.py::GeometricPrimitiveTests::test_reciprocality
25.90s call     clifford/test/test_dg3c.py::BasicTests::test_metric
19.54s call     clifford/test/test_dg3c.py::GeometricPrimitiveTests::test_line
10.66s call     clifford/test/test_tools.py::G3ToolsTests::test_quaternion_conversions
8.79s call     clifford/test/test_tools.py::G3ToolsTests::test_generate_rotation_rotor_and_angle
8.48s call     clifford/test/test_clifford.py::TestBasicAlgebra::test_right_multiplication_matrix[conformal Cl(3)]
8.21s call     clifford/test/test_clifford.py::TestBasicAlgebra::test_left_multiplication_matrix[conformal Cl(3)]
6.08s call     clifford/test/test_io.py::TestHDF5BasicIO::test_write_and_read
6.00s call     clifford/test/test_io.py::TestJSONBasicIO::test_write_and_read
5.97s call     clifford/test/test_io.py::TestHDF5BasicIO::test_write_and_read_array
4.78s call     clifford/test/test_aliases.py::test_import[gac]
4.73s call     clifford/test/test_g3c_tools.py::TestVisualisation::test_draw_objects
2.81s call     clifford/test/test_dpga.py::TestBasicDPGA::test_bivector_identities
2.38s call     clifford/test/test_clifford.py::TestBasicAlgebra::test_right_multiplication_matrix[Cl(4)]
2.26s call     clifford/test/test_clifford.py::TestBasicAlgebra::test_left_multiplication_matrix[Cl(4)]
1.89s call     clifford/test/test_dpga.py::TestBasicDPGA::test_quadric
= 271 passed, 140 skipped, 3 deselected, 8 xfailed, 360 warnings in 1588.20s (0:26:28) =
```

We should either:
* Use a pattern like https://github.com/pygae/clifford/blob/63c50b3198a452aded8f4371276ac25182547c3e/clifford/test/test_g3c_tools.py#L27-L33
* Reduce the number of test iterations if jitting is disabled.

**Comment by eric-wieser** [comment]
#316 does a mixture of both, but the test run is still 4x slower than it was.

**Comment by eric-wieser** [comment]
Resolved in #319 I think.
